src/commands/build.js

```diff
@@ -19,6 +19,8 @@ module.exports.handler = function() {
             configContents[CONSTANTS.CONFIG_CHAINCODES_KEY]
         ).then(() => {
             console.log('Done building!');
+        }).catch((error) => {
+            console.error('Build failed with following error:', error)
```

Comment:
isn't this error catched below?

Reply:
This catch was not needed indeed. The unhandled promise rejection was at another place